rfds/0000-establishing-the-rfd-process.md

```diff
@@ -0,0 +1,440 @@
+may still be comparable to the current more ad-hoc process which often involves
+the production of presentation slides, written documents, and code changes
+which are submitted and shared across QWG meetings and public Pull Requests.
+
+RFDs also do not resolve dysfunction regarding divergence of values or goals
+among the QWG or between CVE and its stakeholders. As the QWG operates by
+consensus, an inability to reach consensus can, despite the RFD process, still
+result in deadlock and an inability to proceed in making changes.
+
+## Compatibility and Migration
+[compatibility-and-migration]: #compatibility-and-migration
+
+There are no compatibility or migration concerns for this RFD, as it concerns
+the establishment of the RFD process itself, and does not contain any proposals
+for amending the CVE Record Format.
+
+## Success Metrics
+[success-metrics]: #success-metrics
+
+6 months after the acceptance and adoption of this RFD process, the QWG should
+solicit a survey to QWG members and outside CVE stakeholders about the value
+of the RFD process. The results of that survey will be scored and compared
+against a threshold, and if the average results of the survey do not pass the
+required minimum threshold to indicate worthwhile value from the process, then
+the RFD process will be rolled back and will not be required for future
+proposals.
+
+To limit the possibility of "gaming" the survey by choosing questions
+after-the-fact which may unduly influence the outcome of the survey, the
+questions to be used are included in this RFD proposal, and shall not be
+changed when the survey is held.
+
+The initial questions on the survey will be used to determine whether answers
+will be considered for scoring. Anyone who answers "false" to both of the
+following questions will not have their answers considered for the purpose of
+scoring the survey results to determine whether the RFD has succeeded at its
+success metric:
+
+1. "I am a participant in the CVE Quality Working Group (have attended at least
+   one CVE QWG meeting in the past six months)"
+2. "I am a CVE ecosystem stakeholder (have consumed CVE records in the past six
+   months)"
+
+All other question answers are given on a 5-level Likert scale, with each
+answer being assigned a point value, as follows:
+
+- Strongly disagree (1 point)
+- Disagree (2 points)
+- Neither agree nor disagree (3 points)
+- Agree (4 points)
+- Strongly agree (5 points)
+
+The questions will be:
+
+3. "The CVE QWG's RFD process has made tracking the status of proposals to
+   amend the CVE Record Format easier."
+4. "The CVE QWG's RFD process has improved the level of rigor applied to
+   proposals to amend the CVE Record Format."
+5. "The CVE QWG's RFD process has made it easier to gather input on proposals
+   to amend the CVE record format from QWG members."
+6. "The CVE QWG's RFD process has made it easier to gathe rinput on proposals
```

Comment:
Minor typo here:

”is easier to gathe rinput” => ”is easier to gather input”

```suggestion
6. "The CVE QWG's RFD process has made it easier to gather input on proposals
```

Reply:
Thanks! Fixed